enterprise/app/jobs/captain/conversation/response_builder_job.rb

```diff
@@ -17,6 +18,22 @@ def perform(conversation, assistant)
 
   delegate :account, :inbox, to: :@conversation
 
+  def detect_conversation_language
+    # Skip if language is already detected
+    # If the account has google translate API enabled, it's possible that the language will already be detected
+    return if @conversation.additional_attributes&.dig('conversation_language').present?
```

Comment:
We could probably add a method in conversation to use conversation.language (would make it easier in the future)

Reply:
Done, also removed language detection as per our discussion